test_scripts/API/SetAudioStreamingIndicator/037_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_media_app_RPC_disallowed_by_user.lua

```diff
@@ -0,0 +1,190 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [USER_DISALLOWED]: Conditions for USER_DISALLOWED result code
+-- [MOBILE_API] SetAudioStreamingIndicator
+-- [HMI_API] [MOBILE_API] AudioStreamingIndicator enum
+-- [PolicyTable] SetAudioStreamingIndicator RPC
+--
+-- Description:
+-- For Genivi applicable ONLY for 'EXTERNAL_PROPRIETARY' Polciies
+-- SDL must return 'USER_DISALLOWED, success:false' to mobile app
+-- in case the requested RPC is included to the group disallowed by the user.
+--
+-- 1. Used preconditions
+-- SDL is built with External Proprietary flag.
+-- Include in Testing_group(user_consent group) RPC SetAudioStreamingIndicator
+-- Register and activate media application
+-- Perform successfull PTU for External Proprietary flow
+-- Disallow by user SetAudioStreamingIndicator
+--
+-- 2. Performed steps
+-- Send SetAudioStreamingIndicator(audioStreamingIndicator = "PAUSE")
+--
+-- Expected result:
+-- SDL->mobile: SetAudioStreamingIndicator_response("USER_DISALLOWED", success:false)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"MEDIA"}
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local json = require('json')
+
+--[[ Local Functions ]]
+
+--[[@create_ptu_SetAudioStreamingIndicator_group: update preloaded_pt and creates PTU files
+! preloaded_pt.json: add Testing_group with RPC SetAudioStreamingIndicator and user consent
+! SetAudioStreamingIndicator_group1.json: Assign groups "Base-4", "Testing_group" to application
+! @parameters: NO
+]]
+local function create_ptu_SetAudioStreamingIndicator_group()
+  commonPreconditions:BackupFile("sdl_preloaded_pt.json")
+  local config_path = commonPreconditions:GetPathToSDL()
+
+  os.execute(" cp " .. config_path .. "sdl_preloaded_pt.json" .. " " .. "SetAudioStreamingIndicator_group1.json" )
```

Comment:
@dboltovskyi, in this way I am creating the file to be sure that when try to open it will exist.
If you insist of any rework, I can create it with "touch <filename>" in case of missing file.